jest.config.integration.js

```diff
@@ -136,7 +136,7 @@ module.exports = {
   // testLocationInResults: false,
 
   // The glob patterns Jest uses to detect test files
-  testMatch: ['**/__tests__/**/*.(integration.test).[jt]s?(x)'],
+  testMatch: ['**/__tests__/**/*.(test).[t]s?(x)', '**/deleteUsers/**/*.(test).[t]s?(x)'],
```

Comment:
How is this different from the `jest.config.js` config?
Why is `deleteUsers` test part of integration tests?

Reply:
CC @utsabc

Reply:
removed typescript test cases from this config